R/input-select.R

```diff
@@ -72,6 +72,10 @@
 #'   }
 #' )
 #' }
+#'
+#' @section Server value:
+#' A vector of character strings, usually with a length of 1, with the value of the selected items.
```

Comment:
If nothing is selected, what happens? (when `multiple=TRUE`, or `Choose an option`=""`)